mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Framework-Test-Class.asciidoc

```diff
@@ -1,7 +1,26 @@
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
 = Test Class
 
 == Overview
 
+In this document we'll show you sample test calsses and how should you set them up.
+
```

Comment:
In this file in section "Annotations" more changes are required. 
Mr. Checker currently supports only JUnit5 and annotations such as "BeforeClass", "AfterClass" do not longer exist. 
So the text with description and code need to be adjusted with proper for JUnit5 annotations. 

The same for section "Sample Setup"

Reply:
I updated both sections and changed the image, please check if everything is ok